### ISSUE TITLE
doublespawning cars if arbitrator switches

### ISSUE THREAD

**Issue by Heady20006** [TanguyOrtegat/esx_jb_stopvehicledespawn#7]
The scripts works grad but if the arbitrator is switching to another all cars in the file are respawning.

**Comment by Heady20006** [comment]
i found out that NetworkIsHost() is the problem isnt it?
If the arbitrator is switchting, we got a new host, the scripts thinks that the cars not spawned and spawning them again.

**Comment by XxFri3ndlyxX** [comment]
Yes this happen with me and my friend too. I disconnected then he told me my car duplicated.

**Comment by TanguyOrtegat** [comment]
the script needs to operate once, thats why I use IsNetworkIsHost()

**Comment by Heady20006** [comment]
But it didnt work. Is there any other natives we can use?

**Comment by XxFri3ndlyxX** [comment]
I have noticed that it does it when you tp far and then tp back to it too. Example. If it go to a door to tp to the weed warehouse location then when done i go back to the location where the car is. The car will duplicate but not all the time though.

**Comment by TanguyOrtegat** [comment]
yea i should sheck the code again, but atm i don't have a lot of time left for fivem :)

**Comment by JaroSound** [comment]
Someone fixed it?

**Comment by TanguyOrtegat** [comment]
don't think so :/ 
and i didn't have the time to look again at the script

**Comment by Presty1** [comment]
If you can fix it that would be awesome I'm willing to use this script. Also I don't know if this is a issue or not. But I got 3 cars out that I own and left the server and came back and one of my vehicles was gone and the other was fixed. It didn't save my vehicles damage or it either fixed itself. Only 1 came back like it should have.

**Comment by JaroSound** [comment]
Is this problem fixed in Latest commit ?

**Comment by TanguyOrtegat** [comment]
i am hoping to find sollutions, but we are getting close
